spec/features/guide_publishing_flow_spec.rb

```diff
@@ -255,10 +255,64 @@
       end
       expect(page).to_not have_button "Publish Guide"
       expect(page).to_not have_button "Send for review"
+      # expect(page).to have_link "View on website", href: [Plek.new.website_root, guide.slug].join('')
+    end
+
+    describe "for a draft guide" do
+      context "that has never been published" do
+        before do
+          guide = create(:guide, :with_draft_edition,
+            slug: "/service-manual/topic-name/new-guide",
+            )
+          visit edit_guide_path(guide)
+        end
+
+        it "has a 'Preview' link" do
+          expect(page).to have_link "Preview", href: "http://draft-origin.dev.gov.uk/service-manual/topic-name/new-guide"
+        end
+
+        it "does not have a 'View on website' button" do
+          expect(page).to_not have_button "View on website"
+        end
+      end
+
+      context "that has just been published" do
+        before do
+          guide = create(:guide, :with_published_edition,
+            slug: "/service-manual/topic-name/just-published",
+            )
+          visit edit_guide_path(guide)
+        end
+
+        it "does not have a 'Preview' link" do
+          expect(page).to_not have_button "Preview"
+        end
+
+        it "has a 'View on website' link" do
+          expect(page).to have_link "View on website", href: "http://www.dev.gov.uk/service-manual/topic-name/just-published"
+        end
+      end
+
+      context "that has one published guide edition" do
+        before do
+          guide = create(:guide, :with_previously_published_edition,
+            slug: "/service-manual/topic-name/published-guide",
+            )
```

Comment:
I tend to outdent closing parentheses

So:

```rb
guide = create(:guide, :with_previously_published_edition,
  slug: "/service-manual/topic-name/published-guide",
)
```

Rather than:


```rb
guide = create(:guide, :with_previously_published_edition,
  slug: "/service-manual/topic-name/published-guide",
  )
```

That might just be personal preference though. Don't know if @thehenster has an opinion on this?